packages/office-ui-fabric-react/src/components/DetailsList/index.ts

```diff
@@ -4,4 +4,3 @@ export * from './DetailsList';
 export * from './DetailsList.types';
 export * from './DetailsRow';
 export * from './DetailsRowCheck';
-export * from './ShimmeredDetailsList';
```

Comment:
@ThomasMichon As a solution you should maybe add a `ShimmeredDetailsList.ts` file at the `packages/office-ui-fabric-react/src` level to serve as an entry point to this component which will contain `export ShimmeredDetailsList from './components/DetailsList/ShimmeredDetailslIst'`. Plus the build is breaking cause I am using this ShimmeredDetailsList in a Shimmer Example page. My solution should fix this build error and give us the result we want regarding the bundle size. I am not an expert when it comes to bundling so I leave this to you...